docs/tutorials/substreams/getting-started.experimental.md

```diff
@@ -0,0 +1,318 @@
+---
+title: 'Getting Started'
+
+order: -9999
+outline: [0,4]
+---
+
+# Substreams Quickstart
+
+The goal of this document is to get a development environment set up in as little time as possible.
+
+Here is a quick introduction to [Substreams](https://substreams.streamingfast.io/).
+
+## Prerequisites
+
+### Installing `helm`
+
+-   [Helm for Windows](https://helm.sh/docs/intro/install/#from-chocolatey-windows).
+-   [Helm for Linux](https://helm.sh/docs/intro/install/#from-script).
+-   [Helm for Mac](https://helm.sh/docs/intro/install/#from-homebrew-macos).
+
+Make sure `helm` is installed properly:
+
+```sh
+helm version
```

Comment:
Output of `helm version` should follow as in the case of `minikube` below, which may very depending on the installed version.

Reply:
added